mochad/L_Mochad1.lua

```diff
@@ -393,7 +393,8 @@ end
 local function rxsec_incoming_data(addr, new_state)
     local trip
     local altit
-    local bcdaddr = convertBCD(addr)
+--    local bcdaddr = convertBCD(addr)
+    local bcdaddr = addr
```

Comment:
yah, Its a good idea to remove that.  A used to have a custom bridge between the old X-10 Home Automation software that ran on Windows.  That software stored everything as BCD.   I don't use that software anymore.